addons/stock/models/stock_quant.py

```diff
@@ -646,6 +646,36 @@ def _compute_complete_name(self):
             res[package.id] = name
         return res
 
+    def _search_location(self, operator, value):
+        if value:
+            packs = self.search([('quant_ids.location_id', operator, value)])
+        else:
+            packs = self.search([('quant_ids', operator, value)])
```

Comment:
If I search on packages whose location_id is not set, I received packages with no quants. It does not seem to be the result I asked. For me packs = self.search([('quant_ids.location_id', operator, value)]) is sufficient in all cases. What do you think about ?